### DISCUSSION TITLE
Compile error,

### DISCUSSION THREAD

**Discussion by zw20045500** [libvips/libvips#2393] [Q&A]
Hi, how to find which so dependcy,  I have install libwebp, but still also say no libwebp. thank you.

where can see the dependcy so name and version.   

![图片](https://user-images.githubusercontent.com/30209916/129202909-48b8e804-44a8-4bbf-b8dc-ea9838c7af7e.png)

![图片](https://user-images.githubusercontent.com/30209916/129202854-60dfe0c3-3e22-4196-889a-79edd15688fc.png)

**Comment by jcupitt** [comment]
Hello, you need the `libwebp-dev` package. It has all the headers the compiler needs.

**Reply by zw20045500** [reply]
hello, I hava install but still the same error.  All third so lib need I to install by self?  can you list all the thrid so full name, it will be useful fow newer ? I find no these info in makefile.

**Reply by zw20045500** [reply]
after yum install , only libwebp and libwebpmux, no libwebpdemux.so. if successful, can take a pic for what is like
![图片](https://user-images.githubusercontent.com/30209916/129304836-260cf336-332c-4d77-8415-d80dbfd795d3.png)

**Comment by jcupitt** [comment]
libvips uses `pkg-config` to find packages. I see:

```
$ pkg-config libwebp libwebpmux libwebpdemux --libs 
-lwebpmux -lwebpdemux -lwebp
```

You're probably missing one of the dev packages. 

I guess you're using redhat, so they'll be called something like `libwebp-devel`.

**Reply by zw20045500** [reply]
I saw the cmake command to find libwebp.  I have Resolved. thank you very much



if test x"$with_libwebp" != x"no"; then
  PKG_CHECK_MODULES(LIBWEBP, libwebp >= 0.6 libwebpmux >= 0.6 libwebpdemux >= 0.6,
    [AC_DEFINE(HAVE_LIBWEBP,1,[define if you have libwebp/libwebpmux/libwebpdemux installed.])
     with_libwebp=yes
     PACKAGES_USED="$PACKAGES_USED libwebp libwebpmux libwebpdemux"
    ],
    [AC_MSG_WARN([libwebp, mux, demux not found; disabling WEBP support])
     with_libwebp=no
    ]
  )
fi
